build.sh

```diff
@@ -477,7 +477,7 @@ generate_NugetPackages()
     echo "DistroRid is "$__DistroRid
     echo "ROOTFS_DIR is "$ROOTFS_DIR
     # Build the packages
-    $__ProjectRoot/run.sh build -Project=$__SourceDir/.nuget/packages.builds -MsBuildLog="/flp:Verbosity=normal;LogFile=$__LogsDir/Nuget_$__BuildOS__$__BuildArch__$__BuildType.log" -BuildTarget -__IntermediatesDir=$__IntermediatesDir -__RootBinDir=$__RootBinDir -BuildNugetPackage=false -UseSharedCompilation=false $__RunArgs $__UnprocessedBuildArgs
+    $__ProjectRoot/run.sh build -Project=$__SourceDir/.nuget/packages.builds -MsBuildEventLogging="/l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log" -MsBuildLog="/flp:Verbosity=normal;LogFile=$__LogsDir/Nuget_$__BuildOS__$__BuildArch__$__BuildType.log" -BuildTarget -__IntermediatesDir=$__IntermediatesDir -__RootBinDir=$__RootBinDir -BuildNugetPackage=false -UseSharedCompilation=false $__RunArgs $__UnprocessedBuildArgs
```

Comment:
What was the need for the addition of the binclash logger as part of this PR?

Reply:
@joperezr Why did we add this? I noticed that we don't use this logger for the linux test build (though we do for the windows test build). Should we enable binclash logging on windows as well?

Reply:
We do have the binclash logger for the product build on Windows already enabled. It is true that we don't have it for tests, and to be honest I can't fully remember why did we need to add this, but I believe it had something to do with the default from config.json not working anymore with the new CLI so we needed to manually specify it instead.

Reply:
Ok, thanks! I ask because binclash logging is causing problems for some linux test build logic I'm working on. I'll probably just disable it selectively.

Reply:
What sorts of problems? If it is hitting issues it is because you do have a bin-clash which is likely a race condition that needs to get fixed.

Reply:
When we build test wrappers, we binplace a dll into the same location for different projects that are built sequentially. We currently only build wrappers on windows, and binclash logging is disabled for this by the -MsBuildEventLogging=" " here: https://github.com/dotnet/coreclr/blob/master/build-test.cmd#L370.

Reply:
I would still caution that you might be asking for trouble if you ever try to run these test builds in parallel.

Reply:
Understood, thanks! I may be able to fix this as I enable the wrapper build on unix.